android/src/main/java/com/equimaps/capacitor_background_geolocation/BackgroundGeolocationService.java

```diff
@@ -82,39 +75,22 @@ void addWatcher(
                 Notification backgroundNotification,
                 float distanceFilter
         ) {
-            FusedLocationProviderClient client = LocationServices.getFusedLocationProviderClient(
-                    BackgroundGeolocationService.this
-            );
-            LocationRequest locationRequest = new LocationRequest();
-            locationRequest.setMaxWaitTime(1000);
-            locationRequest.setInterval(1000);
-            locationRequest.setPriority(LocationRequest.PRIORITY_HIGH_ACCURACY);
```

Comment:
I take it the GPS provider is high accuracy by default?

Reply:
Yup.